src/libpanic_abort/lib.rs

```diff
@@ -53,7 +53,7 @@ pub unsafe extern fn __rust_maybe_catch_panic(f: fn(*mut u8),
 // now hopefully.
 #[no_mangle]
 pub unsafe extern fn __rust_start_panic(_data: usize, _vtable: usize) -> u32 {
-    return abort();
+    abort();
```

Comment:
This is to silence a warning? Do we need to be concerned about back-compat? I guess this just causes new *warnings*, right?